portfolio/src/main/webapp/index.html

```diff
@@ -15,10 +15,10 @@ <h2>My Portfolio</h2>
       </div>
       <img src="/images/Selfie1.jpg" />
 
-    <button onclick="showMessage();">          
-      List of artists
+    <button onclick="showArtist();">          
+      Click here for a random artist
     </button>
-    <p id="message-container"></p>
+    <p id="randomArtist-container"></p>
```

Comment:
keep the naming scheme consistent. Either all camelcase or all dashes. Here it's better to be `random-artist-container`.